pkg/experimentalmetricmetadata/metadata_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright -c Google LLC
```

Comment:
fix header

Reply:
My addlicense expects a year to follow `-y`, so the default is being used (I ran twice and didn't contemplate the error).
```
addlicense:
	@ADDLICENCESEOUT=`$(ADDLICENCESE) -y -c 'OpenTelemetry Authors' $(ALL_SRC) 2>&1`; \
```

```
  -y string
    	copyright year(s) (default "2021")
```